src/main/java/de/aservo/confapi/commons/model/ApplicationLinkBean.java

```diff
@@ -33,22 +41,19 @@
 
     @XmlElement
     @NotNull
-    private ApplicationLinkTypes linkType;
+    private ApplicationLinkTypes type;
```

Comment:
Wie unterscheiden sich denn appType und linkType?

Reply:
@lehmannk Hier aber trotzdem noch die Frage: Wie unterscheiden sich denn appType und linkType?

Reply:
berechtigte Frage. `appType` ist derzeit der ausgelesene `com.atlassian.applinks.api.ApplicationType`als String Repräsentation. `linkType` hingegen ist unser ENUM, dass wir bei setzen des Links verwenden - um zu unterscheiden, welche App überhaupt verlinkt werden soll. Du hast Recht, ist irgendwie doppelt. In der Hoffnung, dass Atlassian die `ApplicationType` nicht irgendwie umbenennt, würde ich für nur `linkType` - oder einfach nur `type` plädieren. Dann parsen wir den String von Atlassian zu unserem Enum.

Reply:
Habe appName rausgenommen, linkName umbenannt.